server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/BaseODCFlowTaskDelegate.java

```diff
@@ -90,6 +112,24 @@ public abstract class BaseODCFlowTaskDelegate<T> extends BaseRuntimeFlowableDele
     private EventBuilder eventBuilder;
     @Autowired
     protected FlowTaskCallBackApprovalService flowTaskCallBackApprovalService;
+    @Autowired
+    protected CloudObjectStorageService cloudObjectStorageService;
+
+    private static final Set<TaskType> SUPPORT_DOWNLOAD_LOG_TASK_TYPES = new HashSet<TaskType>() {
```

Comment:
this set may be useless, we can call `taskService.getLog` to get log content/file directly

Reply:
Yes you are right, done!